_episodes/13-hosting.md

```diff
@@ -42,7 +42,7 @@ communities already using the same service.
 
 As an example, Software Carpentry [is on
 GitHub](https://github.com/swcarpentry/) where you can find the [source for this
```

Comment:
You could also change this one, if you don't mind: `https://github.com/swcarpentry`-> ` {{ swc_github }}` (see again `_config.yml`.